kano_profile/badges.py

```diff
@@ -390,6 +390,31 @@ def save_app_state_variable_with_dialog(app_name, variable, value):
     save_app_state_with_dialog(app_name, data)
 
 
+def update_upwards_with_dialog(app_name, variable, value):
```

Comment:
I'm not fully happy with this name but I suppose that it remains consistent with our other functions and I don't see a better alternative right now. Proceed with my objections noted.